esphome/components/esp32_can/esp32_can.h

```diff
@@ -12,6 +12,8 @@ class ESP32Can : public canbus::Canbus {
  public:
   void set_rx(int rx) { rx_ = rx; }
   void set_tx(int tx) { tx_ = tx; }
+  void set_tx_queue_len(int tx_queue_len) { tx_queue_len_ = optional<int>(tx_queue_len); }
+  void set_rx_queue_len(int rx_queue_len) { rx_queue_len_ = optional<int>(rx_queue_len); }
```

Comment:
Please use the correct types
```suggestion
  void set_tx_queue_len(uint32_t length) { this->tx_queue_len_ = length; }
  void set_rx_queue_len(uint32_t length) { this->rx_queue_len_ = length; }
```

Reply:
good catch, thanks!